### ISSUE TITLE
[addon-docs][ArgsTable] Add show/hide config for default docs page layout

### ISSUE THREAD

**Issue by CTCTIPragg** [storybookjs/storybook#13882]
**Ideal solutions**
Config options for show/hide on the default doc blocks
Example idea:
```
docs: {
  description: { enabled: true, text: `A thing` }
  source: { enabled: true, code: `<foo></foo>`, language: 'html' },
  argsTable: { enabled: false },
  ...
}
```

**Less ideal solutions**
Removal of the 'error' state when no inputs are found, no input components are valid.

**Describe alternatives you've considered**
* Tried using a decorator to remove the no inputs found element but when stories are rendered in iframes the decorator is part of a different document.
* Tried using the `page` parameter of `docs` but this does not appear to be usable out of the box in an Angular project as it requires the use of JSX, it also would cause future improvements/changes to the default story to be lost as it would be manually overridden (edge case sure).

**Are you able to assist to bring the feature to reality?**
possibly but not in any reasonable timeframe

**Additional context**
![image](https://user-images.githubusercontent.com/38093909/107711842-311ef380-6d2d-11eb-8bf4-03b9e7b21e4a.png)

**Comment by shilman** [comment]
Would adding the ability to hide these warnings also satisfy your use case?

**Comment by CTCTIPragg** [comment]
@shilman Yes, hiding warnings would suit the use case I am working with at the moment.

**Comment by digitaltopo** [comment]
Any news on this? Having ` argsTable: { enabled: false }` would be a nice addition. How do these parameters get passed into the docs block components?

**Comment by CTCTIPragg** [comment]
Correct me if I am way off but I believe that parameters are added to somewhat of a global store and addons can consume from that store if they are configured to do so.

I have not had time to look into adding this functionality yet, in our project I have worked around it by adding a no inputs control.
```
export default {
  ...
  argTypes: {
    noInputs: { control: {}, name: 'No inputs' }
  }
}
```

**Comment by valentinpalkovic** [comment]
Thanks for the feature request. The `ArgsTable` block you wanted to disable was removed in Storybook 8 (replaced by `ArgTypes`/`Controls`), and DocsPage is now fully composable via `parameters.docs.page = MyMDXPage` — you literally choose which blocks to render, which is the granular show/hide you wanted. Closing as effectively superseded.
